.github/workflows/commit-workflow-all.yml

```diff
@@ -0,0 +1,369 @@
+
+      - name: build tedge debian package for target
+        uses: actions-rs/cargo@v1
+        # https://github.com/marketplace/actions/rust-cargo
+        with:
+          command: deb
+          args: -p tedge --no-strip --no-build --target=${{ matrix.target }}
+
+      - name: build tedge_mapper debian package for target
+        uses: actions-rs/cargo@v1
+        # https://github.com/marketplace/actions/rust-cargo
+        with:
+          command: deb
+          args: -p tedge_mapper --no-strip --no-build --target=${{ matrix.target }}
+
+      - name: build tedge_agent debian package for target
+        uses: actions-rs/cargo@v1
+        # https://github.com/marketplace/actions/rust-cargo
+        with:
+          command: deb
+          args: -p tedge_agent --no-strip --no-build --target=${{ matrix.target }}
+
+      - name: build tedge_apt_plugin debian package for target
+        uses: actions-rs/cargo@v1
+        # https://github.com/marketplace/actions/rust-cargo
+        with:
+          command: deb
+          args: -p tedge_apt_plugin --no-strip --no-build --target=${{ matrix.target }}
+
+      - name: build examples
+        uses: actions-rs/cargo@v1
+        # https://github.com/marketplace/actions/rust-cargo
+        with:
+          use-cross: true
+          command: build
+          args: --release --examples --target=${{ matrix.target }}
+
+      - name: Strip workaround sawtooth_publisher
+        run: arm-linux-gnueabihf-strip target/${{ matrix.target }}/release/examples/sawtooth_publisher || aarch64-linux-gnu-strip target/${{ matrix.target }}/release/examples/sawtooth_publisher
+
+      - uses: actions/upload-artifact@v2
+        # https://github.com/marketplace/actions/upload-a-build-artifact
+        with:
+          name: tedge_0.3.2_armhf.deb
+          path: target/armv7-unknown-linux-gnueabihf/debian/tedge_0.3.2_armhf.deb
+
+      - uses: actions/upload-artifact@v2
+        # https://github.com/marketplace/actions/upload-a-build-artifact
+        with:
+          name: tedge_apt_plugin_0.3.2_armhf.deb
+          path: target/armv7-unknown-linux-gnueabihf/debian/tedge_apt_plugin_0.3.2_armhf.deb
+
+      - uses: actions/upload-artifact@v2
+        # https://github.com/marketplace/actions/upload-a-build-artifact
+        with:
+          name: tedge_agent_0.3.2_armhf.deb
+          path: target/armv7-unknown-linux-gnueabihf/debian/tedge_agent_0.3.2_armhf.deb
+
+      - uses: actions/upload-artifact@v2
+        # https://github.com/marketplace/actions/upload-a-build-artifact
+        with:
+          name: tedge_mapper_0.3.2_armhf.deb
+          path: target/armv7-unknown-linux-gnueabihf/debian/tedge_mapper_0.3.2_armhf.deb
+
+      - uses: actions/upload-artifact@v2
+        # https://github.com/marketplace/actions/upload-a-build-artifact
+        with:
+          name: sawtooth_publisher
+          path: target/armv7-unknown-linux-gnueabihf/release/examples/sawtooth_publisher
```

Comment:
Also, it would be better that the target arch uses variable like 
```
target/${{ matrix.target }}/release/examples/sawtooth_publisher
```
Copied from https://github.com/thin-edge/thin-edge.io/blob/main/.github/workflows/build-workflow.yml.